cytoolz/itertoolz.pyx

```diff
@@ -963,6 +963,7 @@ cdef class sliding_window:
         cdef Py_ssize_t i
         self.iterseq = iter(seq)
         self.prev = PyTuple_New(n)
+        PyTuple_SET_ITEM(self.prev, i, None)
```

Comment:
Is `i` guaranteed to be `0` here? 
```suggestion
        PyTuple_SET_ITEM(self.prev, 0, None)
```

This is fine, cython will convert the `None` into `Py_None`, which is the C representation of `None`.

Reply:
Got it, i didn't know if it would do the reference counting or not.